pcsx2/DEV9/DEV9Config.cpp

```diff
@@ -0,0 +1,114 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2020  PCSX2 Dev Team
```

Comment:
2021 for new files, there are others as well.

Reply:
I believe I've corrected all the copyright headers on all the new files

Reply:
![image](https://user-images.githubusercontent.com/24227051/111713740-7f557400-8850-11eb-86b8-25fd1b3e25eb.png)
Just this one left.

Reply:
Ah, just took the line `2021 for new files` and didn't look at the file the comment was made on.
I did also notice I missed another file, but that change I squashed.

Reply:
Actually, I got confused by the two similarly named files, I'll amend that tomorrow

Reply:
Okay, _now_ we should be good